cmd/pj-rehearse/main.go

```diff
@@ -101,19 +90,35 @@ func (o *options) validate() error {
 	logrus.SetLevel(level)
 
 	errs = append(errs, o.kubernetesOptions.Validate(o.dryRun))
+	errs = append(errs, o.githubEventServerOptions.DefaultAndValidate())
+	errs = append(errs, o.github.Validate(o.dryRun))
 
-	if o.serverMode {
-		errs = append(errs, o.githubEventServerOptions.DefaultAndValidate())
-		errs = append(errs, o.github.Validate(o.dryRun))
-	} else {
-		if len(o.releaseRepoPath) == 0 {
-			errs = append(errs, errors.New("--candidate-path was not provided"))
-		}
+	if o.dryRun {
+		errs = append(errs, o.dryRunOptions.validate())
 	}
 
 	return utilerrors.NewAggregate(errs)
 }
 
+type dryRunOptions struct {
```

Comment:
can we move those on `pkg/rehearse/options.go?`

Reply:
Do you mean that I should create a new file to move these options to? What do we get out of that move?

Reply:
I could just move these into the `options` struct directly, but I thought having a separate struct made it more clear that they were only for the test.